wms_to_mfm_openapi_spec.yaml

```diff
@@ -270,6 +297,111 @@ paths:
       summary: Create an Order. (Required)
       tags:
       - Orders
+  /api/v1/wms/requests:
+    get:
+      description: |-
+        Return a list of request status which are still opened for the robot, or were teriminated by the robot, and need to be cleared by the WMS (cf. POST /clear endpoint).
+
+        A request status is composed of:
+
+        - an order_id
+        - an request_id
+        - a request state (created, started, grasped, picked, completed, aborted)
+
+        Response:
+        - 200
+      operationId: mfm_rest_api_wms_api_order_requests_status
+      parameters: []
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/RequestStatus'
+              examples:
+                RequestStatusList:
+                  value:
+                    - order_id: "ORDER_01"
+                      request_id: "REQUEST_01"
+                      status: "picked"
+                    - order_id: "ORDER_01"
+                      request_id: "REQUEST_02"
+                      status: "completed"
+                    - order_id: "ORDER_03"
+                      request_id: "REQUEST_01"
+                      status: "finished"
+                      custom_fields:
+                        customKey: customValue
+        400:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/FailureResponse'
+              examples:
+                Order id already exists:
+                  value:
+                    error: "Order with order_id ORDER_01 already exists"
```

Comment:
The error doesn't really apply here to this endpoint.
I also can't think of an error that we would expect here ... besides we are down.